frontend/src/layouts/Footer.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import styled from 'styled-components';
+import { useTenant } from 'tenants';
+
+/**
+ * Footer component that applies tenant environmental styling.
+ * @param param0 Footer properties.
+ * @returns Footer component.
+ */
+export const Footer: React.FC<React.HTMLAttributes<HTMLElement>> = ({ ...rest }) => {
+  const tenant = useTenant();
+  return <FooterStyled {...rest} backgroundColor={tenant.colour}></FooterStyled>;
+};
+
+interface IFooterProps {
+  // Background color of footer.
+  backgroundColor: string;
+}
+
+const FooterStyled = styled.footer<IFooterProps>`
```

Comment:
didn't HeaderStyled get extracted to its own component?

Reply:
or is there two HeaderStyled components maybe?

Reply:
Regrettably the existing structure has multiple components that are "headers".  This particular one and the footer are literal "header" HTML elements for the layout.  I didn't extract the styled component here because it isn't used anywhere else.